doc/userguide.rst

```diff
@@ -581,7 +583,7 @@ The outputs of the function are the time step, the average value of the
 constant of motion, and its RMSD during the simulation. The fourth
 column gives the measured slope of the constant of motion - a large
 value here would indicate a strong drift and hence a problem in the
-integrator. Even without strong drift, as in the current situation, a
+integrator. Even without a strong drift, as in the current situation, a
```

Comment:
I don't feel this improves the sentence. If you think it needs improvement, maybe something like "Even when the constant of motion does not exhibit strong drift, as in the current ..." would make it more clear?

Reply:
I just reverted the change.